src/lib/openapi/spec/playground-feature-schema.test.ts

```diff
@@ -1,23 +1,134 @@
+                ...constraint,
+                result,
+            }));
+
+const playgroundStrategyConstraints = (): Arbitrary<
+    PlaygroundConstraintSchema[]
+> => fc.array(playgroundStrategyConstraint());
+
+const playgroundSegment = (): Arbitrary<PlaygroundSegmentSchema> =>
+    fc.record({
+        name: fc.string({ minLength: 1 }),
+        id: fc.nat(),
+        result: fc.boolean(),
+        constraints: playgroundStrategyConstraints(),
+    });
+
+const playgroundStrategy = (
+    name: string,
+    parameters: Arbitrary<Record<string, string>>,
+): Arbitrary<PlaygroundStrategySchema> =>
+    fc.record({
+        name: fc.constant(name),
+        result: fc.oneof(
+            fc.record({
+                evaluationStatus: fc.constant(
+                    playgroundStrategyEvaluation.evaluationComplete,
+                ),
+                enabled: fc.boolean(),
+            }),
+            fc.record({
+                evaluationStatus: fc.constant(
+                    playgroundStrategyEvaluation.evaluationIncomplete,
+                ),
+                // reason: fc.constantFrom(
+                //     ...playgroundStrategyEvaluation.incompleteEvaluationCauses,
+                // ),
+                enabled: fc.constantFrom(
+                    playgroundStrategyEvaluation.unknownResult,
+                    false as false,
+                ),
+            }),
+        ),
+        parameters,
+        constraints: playgroundStrategyConstraints(),
+        segments: fc.array(playgroundSegment()),
+    });
+
+const playgroundStrategies = (): Arbitrary<PlaygroundStrategySchema[]> =>
+    fc.array(
+        fc.oneof(
+            playgroundStrategy('default', fc.constant({})),
+            playgroundStrategy(
+                'flexibleRollout',
+                fc.record({
+                    groupId: fc.lorem({ maxCount: 1 }),
+                    rollout: fc.nat({ max: 100 }).map(String),
+                    stickiness: fc.constantFrom(
+                        'default',
+                        'userId',
+                        'sessionId',
+                    ),
+                }),
+            ),
+            playgroundStrategy(
+                'applicationHostname',
+                fc.record({
+                    hostNames: fc
+                        .uniqueArray(fc.domain())
+                        .map((domains) => domains.join(',')),
+                }),
+            ),
+
+            playgroundStrategy(
+                'userWithId',
+                fc.record({
+                    userIds: fc
+                        .uniqueArray(fc.emailAddress())
+                        .map((ids) => ids.join(',')),
+                }),
+            ),
+            playgroundStrategy(
+                'remoteAddress',
+                fc.record({
+                    IPs: fc.uniqueArray(fc.ipV4()).map((ips) => ips.join(',')),
+                }),
+            ),
+        ),
+    );
```

Comment:
This file contains arbitraries for all the playground feature subschemas for because it tests the entire deep schema (and because it used to all be one big file).

I'd be happy to extract the arbitraries into test files for the individual schemas, but I'm not sure it's necessary.